packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -498,9 +451,7 @@ exports.createNodesForContentType = ({
       let entryNode = {
         id: mId(space.sys.id, entryItem.sys.id, entryItem.sys.type),
         spaceId: space.sys.id,
-        contentful_id: process.env.EXPERIMENTAL_CONTENTFUL_SKIP_NORMALIZE_IDS
-          ? entryItem.sys.id
-          : entryItem.sys.contentful_id,
+        contentful_id: entryItem.sys.id,
```

Comment:
Tried dropping this entirely but that does break some tests so let's keep it for now. I think it makes sense for there to be a `contentful_id` to recognize atomic ids for content from Contentful

Reply:
yes, we need to be able to identify content via contentful_id  👍